src/Router.js

```diff
@@ -18,6 +19,7 @@ const Router = () => {
         <Route path="/detail" element={<Detail />} />
         <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<SignUp />} />
+        <Route path="/footer" element={<Footer />} />
       </Routes>
```

Comment:
```suggestion
      </Routes>
      <Footer />
```
footer는 경로에 따라 보여주는 것 보다는
모든 페이지에서 보여지는 것이 옳을 것 같네요!